packages/snaps-simulation/src/helpers.ts

```diff
@@ -111,6 +111,16 @@ export type SnapHelpers = {
    */
   runCronjob(cronjob: CronjobOptions): SnapRequest;
 
+  /**
+   * Run a background event in the snap. This is similar to {@link request}, but the
+   * request will be sent to the `onCronjob` method of the snap.
+   *
+   * @param backgroundEvent - The background event request. This is similar to a JSON-RPC
+   * request, and is normally specified in the `request` param of the `snap_scheduleBackgroundEvent` method.
+   * @returns The response promise, with extra {@link SnapRequestObject} fields.
+   */
+  runBackgroundEvent(backgroundEvent: CronjobOptions): SnapRequest;
```

Comment:
```suggestion
  onBackgroundEvent(backgroundEvent: CronjobOptions): SnapRequest;
```

Reply:
I guess the weird thing about this is that background events really use `onCronjob` so `onBackgroundEvent` doesn't exist on the Snap implementation side at least 🤔

Reply:
Yeah that's what I was thinking @FrederikBolding

Reply:
I think of it more as it emitting a background event.

Reply:
Yeah, that makes sense as well. Changed it.